src/main/java/carter/Storage.java

```diff
@@ -0,0 +1,97 @@
+package carter;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represent the storage for the task in Carter.
+ */
+public class Storage {
+    private final String filePath;
+
+    /**
+     * Constructs a new instance of Storage with specified file path.
+     *
+     * @param filePath The file path where the tasks to be stored.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Load the tasks from the storage file.
+     * If the file does not exist, it will be created.
+     *
+     * @return A list of tasks from storage file.
+     * @throws CarterException If the file is corrupted.
+     */
+    public List<Task> load() throws CarterException {
```

Comment:
Good javadoc comments, code is easily understandable and i can understand the Storage class easily